core/src/test/resources/after-org.jboss.arquillian.persistence.core.data.provider.SqlScriptProviderForScriptsAppliedAfterTestMethodTest$ApplyScriptAfterAnnotatedClass.sql

```diff
@@ -1,2 +1,2 @@
-INSERT INTO useraccount (id, firstname, lastname, username, password) VALUES (1, 'John', 'Smith', 'doovde', 'password');
-INSERT INTO useraccount (id, firstname, lastname, username, password) VALUES (2, 'Clark', 'Kent', 'superman', 'kryptonite');
+INSERT INTO useraccount (id, firstname, lastname, username, password) VALUES (1, 'John', 'Smith', 'doovde', 'password');
+INSERT INTO useraccount (id, firstname, lastname, username, password) VALUES (2, 'Clark', 'Kent', 'superman', 'kryptonite');
```

Comment:
Is it line-ending change b/c of windows? If so I would suggest to adjust your IDE and re-format to linux line ending :)